pkg/fedora/tarsnap.spec

```diff
@@ -0,0 +1,59 @@
+Name:           tarsnap
+Version:        1.0.36.1
+Release:        2%{?dist}
+Summary:        Secure, efficient online backup service
+
+License:        Proprietary
+URL:            http://www.tarsnap.com/
+Source0:        http://www.tarsnap.com/download/tarsnap-autoconf-%{version}.tgz
+
+BuildRequires:  gcc
+BuildRequires:  pkgconfig(bash-completion)
+BuildRequires:  pkgconfig(bzip2)
+BuildRequires:  pkgconfig(ext2fs)
+BuildRequires:  pkgconfig(libcrypto)
```

Comment:
As a general note, how did you decide on this list of requirements?  The Tarsnap website claims that

```
sudo yum install gcc make glibc-devel openssl-devel zlib-devel e2fsprogs-devel
```

works, and I checked that on a bare CentOS installation.  Does Fedora have very different package names, or should we be fixing something in the website list for CentOS users, or...?